Apps/phwinrm/parse_callbacks.py

```diff
@@ -100,22 +112,43 @@ def list_processes(action_result, response):
         )
 
     output = response.std_out
-    lines = output.splitlines()
-    for line in lines[2:]:
-        process = {}
+    lines = [y for y in [i for i in output.splitlines() if i] if not y.startswith('-----')]
+    column_headers = lines[0]
+    column_headers_list = column_headers.split()
+    column_indexes = get_column_header_indexes(column_headers)
+
+    header = [y.lower() for y in (column_headers_list + [f"unknown_column_{x}" for x in range(10)])]
```

Comment:
instead of arbitrarily appending 10 unknown column headers to the list can we instead just do a length check in

```
     for i, x in enumerate(columns):
            process_dict[header[i]] = x
```

which sets the key in process_dict to `unknown_column_{i}` if `i >= len(header)`

Reply:
Fixed